dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -135,6 +133,8 @@ public void runWESCommand(String jsonString, File localPrimaryDescriptorFile, Fi
         String workflowTypeVersion = WORKFLOW_TYPE_VERSION;
         if ("CWL".equals(languageType.toUpperCase())) {
             workflowTypeVersion = "v" + WORKFLOW_TYPE_VERSION;
+        } else if ("WDL".equals(languageType.toUpperCase())) {
```

Comment:
check if there's an enum for "WDL", possibly use equalsignorecase

Reply:
There should be one for all languages.

Reply:
used equalsIgnoreCase